### ISSUE TITLE
[Docs - New Feature]: configuring VS Code on Desktop as default IDE

### ISSUE THREAD

**Issue by akosyakov** [gitpod-io/website#1262]
### Describe your feature

After https://github.com/gitpod-io/gitpod/pull/6671 is merged. We should update docs to explain how to configure VS Code (Insiders) on Desktop by default.

### How to test

Go to settings, select enable desktop IDEs, choose VS Code. Then start a workspace and make sure that VS Code on Desktop is opened.

**Comment by loujaybee** [comment]
Makes sense, will remove it from groundwork for now (not sure I remember doing that) but it's currently in inbox. 

So removing, it's attached to VS Code desktop GA.
